templates/zerver/help/change-your-organizations-name.md

```diff
@@ -1,7 +1,7 @@
 # Change your organization's name
 Only an administrator can change the name of the organization.
 
-Follow these steps to change the name of the organization
+{!follow-steps.md!} change the name of the organization
```

Comment:
`to change` instead of just `change`?
If instead your contents are correct, you may want to look at change-a-users-name.md line 6.